doc/classes/Engine.xml

```diff
@@ -83,11 +91,24 @@
 			<description>
 				Returns the total number of frames passed since engine initialization which is advanced on each [b]physics frame[/b]. See also [method get_process_frames].
 				[method get_physics_frames] can be used to run expensive logic less often without relying on a [Timer]:
-				[codeblock]
+				[codeblocks]
+				[gdscript]
 				func _physics_process(_delta):
 				    if Engine.get_physics_frames() % 2 == 0:
 				        pass  # Run expensive logic only once every 2 physics frames here.
-				[/codeblock]
+				[/gdscript]
+				[csharp]
+				public override void _PhysicsProcess(double delta)
+				{
+				    base._PhysicsProcess(delta);
```

Comment:
My logic for the base calls: we have them in the script templates in the editor, so I leave them in the documentation. But happy to remove if need be.

Reply:
Yeah, it's probably a good idea to have them. I don't know if we want to remove them since they are not the focus of the example but I was just bringing attention to the inconsistency with the GDScript example.

Maybe someone from @godotengine/documentation can clarify what should be done in these cases.

Reply:
I asked that question in #documentation on 🚀 and got no reply. So I'm not sure what you'd rather have me do.

Reply:
I'm ok leaving `base` in.